configurator/.terraform/environment

```diff
@@ -0,0 +1 @@
+gs-test-ecs
```

Comment:
have you added this on purpose?

Reply:
pff not at all, thanks I will remove this